src/os/bluestore/BlueStore.cc

```diff
@@ -17551,7 +17559,8 @@ void BlueStore::read_allocation_from_single_onode(
 
       // skip repeating extents
```

Comment:
I think there are 2 reasonable long term solutions:
1) make offset check complete (preferred)
2) drop it altogether